techniques/ios/MASTG-TECH-0090.md

```diff
@@ -5,17 +5,55 @@ platform: ios
 
 If you want to enable dynamic testing with Frida but don't have access to a jailbroken device, you can patch and repackage the target app to load the [Frida gadget](https://www.frida.re/docs/gadget/). This way, you can instrument the app and do everything you need to do for dynamic analysis (of course, you can't break out of the sandbox this way). However, this technique only works if the app binary isn't FairPlay-encrypted (i.e., obtained from the App Store).
 
-The easiest way to inject Frida into an installed application is by using frida-server. However, if this is not possible, the Frida Gadget can be injected into a decrypted IPA file.
+On a jailbroken device, you can run `frida-server` which will take care of the injection for you, even in encrypted apps. However, on a non-jailbroken device we have to manually prepare the application. There are two approaches we can take:
+
+- Install a debug version of the application and inject during application launch
+- Repackage the application to already include the Frida Gadget
 
 As an alternative to this automated approach, see @MASTG-TECH-0091.
 
-## @MASTG-TOOL-0118
+You can inject Frida into an application using @MASTG-TOOL-0039, @MASTG-TOOL-0118 or @MASTG-TOOL-0038
```

Comment:
The ID MASTG-TOOL-0039 is correct for Frida, but we don't explain how to inject Frida into an app running on a non-jailbroken device. Shall we link to the Frida documentation instead?

```suggestion
You can inject Frida into an application using @MASTG-TOOL-0118 or @MASTG-TOOL-0038, or follow the [instructions from the Frida website](https://frida.re/docs/ios/#without-jailbreak).
```

Reply:
I've removed tool 39 and the link to the frida website. This technique is explaining how to do it on a non-jailbroken, so would be weird to link to the frida docs this way.